lib/Doctrine/Connection/Sqlite.php

```diff
@@ -87,8 +93,18 @@ public function connect()
             return false;
```

Comment:
The method is annotated to return void, let's not return a boolean :)
```suggestion
            return;
```

Reply:
Sadly, it is the PHP Doc that is wrong here.

The interface to respect is defined on [Doctrine_Connection::connect()](https://github.com/FriendsOfSymfony1/doctrine1/blob/master/lib/Doctrine/Connection.php#L455)

Reply:
908bcbb316aa372ed887a4973121af5b8d890ee7 I have set a "@return boolean", exactly like the abstract Doctrine_Connection::connect()

is it ok ?

Reply:
Hi @thePanz, you've given a thumbs up to the change from @Tybaze therefore it should be ok to accept this pr, shouldn't it?